File: Level1/문자열 내 p와 y의 개수/Solution.js
function solution(s){
  let alphaPCount = 0;
  let alphaYCount = 0;
  
  for (let i = 0; i < s.length; i++) {
    if (s.charAt(i).toLowerCase() === 'y') {
      ++alphaYCount;
    } else if (s.charAt(i).toLowerCase() === 'p') {
      ++alphaPCount;
    }
  }
  
  if (alphaPCount === alphaYCount) {
    return true;
  } else {
    return false;
  }
}

File: Level1/문자열 내 p와 y의 개수/Solution.java
class Solution {
  boolean solution(String s) {
    int pCount = 0;
    int yCount = 0;

    for (int i = 0; i < s.length(); i++) {
      String alpha = String.valueOf(s.charAt(i)).toUpperCase();

      if ("Y".equals(alpha)) {
        yCount++;
      } else if ("P".equals(alpha)) {
        pCount++;
      }
    }

    if (pCount == yCount) {
      return true;
    }

    return false;
  }
}